### DISCUSSION TITLE
Certain Characters in Prompt Not Appearing Properly

### DISCUSSION THREAD

**Discussion by aaron-kwk** [JanDeDobbeleer/oh-my-posh#3035] [Troubleshoot]
I just installed OhMyPosh following a tutorial online, but some of the characters for the prompt appear as empty boxes rather than what they're supposed to be:

![image](https://user-images.githubusercontent.com/70349283/199326382-612d14c0-c1e2-407d-8991-b51b1008bab6.png)

I'm using Cascadia Mono like he does, but if this is a font issue are there fonts that may fix this issue?

For reference, the theme I'm trying to use is the out-of-the-box theme "takuya".

**Comment by JanDeDobbeleer** [comment]
@kim3470 yes, we even have a [section](https://ohmyposh.dev/docs/installation/fonts) for that.

**Reply by aaron-kwk** [reply]
My bad, thanks for replying
